Makefile

```diff
@@ -14,9 +14,11 @@ NPM_PKG_SPECS = \
 
 PATH:=$(shell pwd)/$(TOOLS_BIN):$(PATH)
 
+OSTYPE:=$(shell uname -msr)
```

Comment:
`OSTYPE` is an unexported shell builtin, and this is my mistake. Using uname for this might not be stable output.

`OSTYPE:=$(shell echo $${OSTYPE})` is a bit roundabout, but works for me and might be worth a test.

This should output lowercase `darwin*` on macos.